Scribe-i18n/scripts/iOS/convert_xcstrings_to_jsons.py

```diff
@@ -38,8 +55,11 @@
 
         data[key] = translation
 
-    json.dump(data, dest, indent=2, ensure_ascii=False)
-    dest.write("\n")
+    # Write to the destination JSON file using a context manager
+    with open(f"{directory}/{lang}.json", "w") as dest:
```

Comment:
Same thing as above goes here, I think it should be `f"{directory}/jsons/{lang}.json"`.